### DISCUSSION TITLE
v0.12.13 - The Christmas Islands Update

### DISCUSSION THREAD

**Discussion by ThePotatoGuy** [Monika-After-Story/MonikaModDev#9799] [Announcements & Releases]
_because the north pole is in desperate need of land_

# Gameplay Changes

* new d25 islands content

* punctuation can no longer be used in player names

# Fixes

* typofixes

* fixed certain bday events from repeating too much

* fixed hangman unlock dialogue not making sense with new games

## What's Changed
* D25 additions to the island event by @Booplicate in https://github.com/Monika-After-Story/MonikaModDev/pull/9796
* Fix 922 nts bug by @jmwall24 in https://github.com/Monika-After-Story/MonikaModDev/pull/9787
* Better hangman unlock dlg paths by @jmwall24 in https://github.com/Monika-After-Story/MonikaModDev/pull/9701
* Replace special Windows characters in path name (fix #9557) by @dreamscached in https://github.com/Monika-After-Story/MonikaModDev/pull/9582
* Fypo Tixes by @Booplicate in https://github.com/Monika-After-Story/MonikaModDev/pull/9733

**Full Changelog**: https://github.com/Monika-After-Story/MonikaModDev/compare/v0.12.12...v0.12.13

<hr /><em>This discussion was created from the release <a href='https://github.com/Monika-After-Story/MonikaModDev/releases/tag/v0.12.13'>v0.12.13 - The Christmas Islands Update</a>.</em>

**Comment by Dewdrop730** [comment]
Yippee! Thanks for updating the game! Have a merry Christmas!
